styles/dialog/fight.scss

```diff
@@ -101,37 +181,38 @@
                 color: black;
                 border-radius: 3px;
             }
-
-            .remove-fighter {
-                width: 1.5em;
-                height: 1.5em;
-                padding: 1px;
-                line-height: 1.3em;
-                color: #800;
-            }
-
-            .actor-name {
-                font-size: 1.125em;
-                font-weight: bold;
-                width: calc(100% - 1.5em);
-                &:hover {
+     
+            .fighter-action-panel {
+                display: flex;
+                flex-direction: row;
+                .remove-fighter {
+                    width: 1.5em;
+                    height: 1.5em;
+                    padding: 1px;
+                    line-height: 1.3em;
                     color: #800;
                 }
-            }
-            .portrait {
-                width: 80px; height: 80px;
-                border: 0;
-                object-fit: cover;
-            }
-            .details {
-                width: calc(95% - 80px);
-                margin-left: 2%;
-                select {
-                    width: 100%;
+                .actor-name {
+                    display: flex;
+                    justify-content: center;
+                    align-items: center;
```

Comment:
```suggestion
                    text-align: center;
```

Reply:
![image](https://user-images.githubusercontent.com/10693583/104725381-dde26280-56e6-11eb-862c-c50d3160feeb.png)

Reply:
I'm not seeing that at all.

![image](https://user-images.githubusercontent.com/5915880/104750784-ca98bc80-5711-11eb-9274-de1f88c0b91e.png)
![image](https://user-images.githubusercontent.com/5915880/104750811-d2586100-5711-11eb-8563-cdfc3323df02.png)